### PR TITLE
MOTS-85: Add voto info to exception

### PR DESCRIPTION
**Pull request by dpiaseckisoldevelo**

### CODE REVIEW

File: src/main/java/org/motechproject/mots/service/ModuleAssignmentService.java [hunk 1]
```diff
@@ -61,7 +61,8 @@ public void assignModules(AssignedModules assignedModules) {
       ivrService.addSubscriberToGroups(ivrId,
           getIvrGroupsFromModules(existingAssignedModules.getModules()));
     } catch (IvrException ex) {
-      String message = "Could assign module for CHW, because of IVR module assignment error";
+      String message = "Could assign module for CHW, because of IVR module assignment error. "
+          + ex.getClearVotoInfo();
```
Comment (kkaczmarczyk) [inline src/main/java/org/motechproject/mots/service/ModuleAssignmentService.java]: I would just add a new line before ex.getClearVotoInfo() to make it more readable

File: src/main/java/org/motechproject/mots/service/CommunityHealthWorkerService.java [hunk 2]
```diff
@@ -40,7 +40,8 @@ public CommunityHealthWorker createHealthWorker(CommunityHealthWorker healthWork
       String ivrId = ivrService.createSubscriber(phoneNumber, name);
       healthWorker.setIvrId(ivrId);
     } catch (IvrException ex) {
-      String message = "Could not create CHW, because of IVR subscriber creation error";
+      String message = "Could not create CHW, because of IVR subscriber creation error. "
+          + ex.getClearVotoInfo();
```
Comment (kkaczmarczyk) [inline src/main/java/org/motechproject/mots/service/CommunityHealthWorkerService.java]: Same here, with new line exception message probably would look better

File: src/webapp/js/utils/api-client.js [hunk 3]
```diff
@@ -30,12 +30,17 @@ const handleError = (error) => {
 
       dispatch(signoutUser());
       break;
-    default:
-      if (error.response.data.message) {
-        Alert.error(error.response.data.message);
+    default: {
+      const errorMessage = error.response.data.message;
+      if (errorMessage) {
+        const alertTimeout = Math.max(5000, errorMessage.length * 100);
```
Comment (kkaczmarczyk) [inline src/webapp/js/utils/api-client.js]: :+1:

**Comment by dpiaseckisoldevelo** [comment]
@kkaczmarczyk I've added extra space between errors.
zaznaczenie_034

**Pull request merged**